examples/express/client.js

```diff
@@ -14,10 +14,10 @@ function makeRequest() {
   tracer.withSpan(span, async () => {
     try {
       const res = await axios.get('http://localhost:8080/run_test');
-      span.setStatus({ code: api.CanonicalCode.OK });
+      span.setStatus({ code: api.StatusCode.OK });
```

Comment:
This should be removed according to [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-status):

```
Generally, Instrumentation Libraries SHOULD NOT set the status code to Ok, unless explicitly configured to do so. Instrumention libraries SHOULD leave the status code as Unset unless there is an error, as described above.
```

Reply:
but this is just example, sowing how to change status, which by default is unset

Reply:
Cool, I didn't see it was an example.
Setting status code to OK should still be removed in many other places in plugins code though.